protocol/messenger.go

```diff
@@ -914,6 +921,26 @@ func (m *Messenger) Start() (*MessengerResponse, error) {
 		}
 	}
 
+	if newsFeedFeatureFlag {
+		myCallback := func(item *gofeed.Item) {
+			fmt.Println("Received new item:", item.Title)
```

Comment:
I know it's under a feature flag, but please use `logger` 🙏

Reply:
I see more `fmt.` below as well
We should pass the `logger` into the news feed manager

Reply:
ok I see you have a `TODO use logger`. Up yo you, but I'd better use it right away

Reply:
I did that in this other PR https://github.com/status-im/status-go/pull/6487